recipe/meta.yaml

```diff
@@ -1,17 +1,19 @@
 {% set name = "antlr" %}
-{% set version = "4.8" %}
-{% set sha256 = "" %}
+{% set major_version = "4" %}
+{% set minor_version = "8" %}
+{% set version = major_version + "." + minor_version %}
+{% set revision = "1" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://github.com/{{ name }}/{{ name }}{{ version[0] }}/archive/{{ version }}.tar.gz
-  sha256: c8e41fa04e4d79ec2dc91dad9ddfd202b09b50fd59f17f3a10f52dbdffdaf70d
+  url: https://github.com/{{ name }}/{{ name }}{{ major_version }}/archive/{{ version }}.tar.gz
+  sha256: 992d52444b81ed75e52ea62f9f38ecb7652d5ce2a2130af143912b3042a6d77e
```

Comment:
yeah, i'm grumpy about the sha changes...

Reply:
Me too...